apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -616,6 +636,11 @@ defmodule OMG.ChildChain.BlockQueue.Core do
     :ok
   end
 
+  defp log_tx_underpriced(submission) do
+    _ = Logger.debug("Submission #{inspect(submission)} got rejected due to low gas price - ignored")
```

Comment:
I again don't think debug log brings much value (probably 0 in this case as this would probably only happen on mainnet env).

Reply:
whoops

Reply:
d7b5d12